src/test/java/StorageTest.java

```diff
@@ -0,0 +1,54 @@
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.DirectoryStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import duke.Duke;
+import duke.Storage;
+import duke.TaskList;
+
+public class StorageTest {
```

Comment:
Perhaps a short description can be provided to explain what the tests are about.